chrome-extension/src/index.js

```diff
@@ -77,11 +77,7 @@ if (!window.location.ancestorOrigins.contains(extensionOrigin) && !document.getE
       /* "_2aha" is unique class name of btn for media content */
       /* "_5f0n" is class name of container of "_2aha" */
       if (document.querySelector('._5f0n') && !document.querySelector('#codeShareBtn')) {
```

Comment:
I guess _5f0n is generated by the build system.
So, It cloud changed each build.

You need to choose a sticky value.
Ex: expanded-sprout-list